internal/docker/docker.go

```diff
@@ -301,3 +334,47 @@ func writeRequest(state *config.LibrarianState, libraryID, jsonFilePath string)
 
 	return nil
 }
+
+// readResponse reads the library state from configure-response.json.
+//
+// The response file is removed afterwards.
+func readResponse(contentLoader func(data []byte, state *config.LibraryState) error, jsonFilePath string) (*config.LibraryState, error) {
```

Comment:
Can `readResponse` and `writeLibrarianState` move to state.go to centralize operations on state.yaml and state struct?

Reply:
I discussed a similar issue with Cody in https://github.com/googleapis/librarian/pull/1017/files#r2240554391, I decided to move these functions to `state.go`.